astromon/db.py

```diff
@@ -515,7 +519,7 @@ def set_formats(dat):
             dat[col.name].info.format = fmts.get(col.name, '.2f')
 
 
-def get_cross_matches(name='astromon_21', dbfile=None):
+def get_cross_matches(name='astromon_21', dbfile=None, **kwargs):
     """
     Get a standard cross-match of observations, x-ray sources and catalog counterparts in `dbfile`.
```

Comment:
I could not find the actual filter values of the standard cross-matches anywhere in the documentation. This is important information.